guide-template.md

```diff
@@ -1,15 +1,55 @@
+<!-- Please use the template below when creating a guide. All fields should be in YAML frontmatter -->
+
 ---
 title: "<Guide Name>"
+permalink: <Guide Name>/
 excerpt: "One sentence description of guide"
-creationDate: 2016-11-03T10:01:43-04:00
-modifiedDate: 2016-11-03T10:01:43-04:00
+date: 2016-11-03
+<!-- Author Name is optional-->
+author: "<Author Name>"
+<!-- modifiedDate is optional, should only be added if there is an update to the guide  -->
+modifiedDate: 2016-12-04
+layout: single
 ---
 
 <!-- Do not need a title because it is added in the frontmatter -->
 
+<!--Include brief, optional intro for the guide here.-->
+
 <!-- This will add a Table of Contents -->
 {% include toc %}
 
-## Guide Section Header
 
 ## Guide Section Header
+<!-- Include section content here -->
+
+## Guide Section Header 2
+<!-- Include section content here -->
+
+## Guide Section Header 3
+<!-- Include section content here -->
+
+
+------------------------------------------------------------
```

Comment:
I think we cut this just to reduce clutter and maybe in the fron <!--- ---> we throw in a link to the .md of a guide?